examples/ucp_hello_world.c

```diff
@@ -67,18 +67,25 @@ enum ucp_test_mode_t {
     TEST_MODE_EVENTFD
 } ucp_test_mode = TEST_MODE_PROBE;
 
+typedef enum {
+    FAILURE_MODE_NONE,
+    FAILURE_MODE_SEND,
+    FAILURE_MODE_RECV,
+    FAILURE_MODE_KEEPALIVE
+} ucp_failure_mode_t;
```

Comment:
`failure_mode_t`
otherwise - it means that this is defined in UCP

Reply:
hmmm, named way same as ucp_test_mode_t...
ok, renamed